launcher/src/backend/NodeConnection.int.js

```diff
@@ -0,0 +1,69 @@
+import { HetznerServer } from "./HetznerServer";
+import { NodeConnection } from "./NodeConnection";
+import { nodeOS } from "./NodeOS";
+const log = require('electron-log');
+
+
+jest.setTimeout(500000);
+
+function Sleep(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+test('prepareStereumNode on ubuntu', async () => {
+
+    const serverSettings = {
+        name: 'my-server',
```

Comment:
Format for server name is with molecule: `<ansible-role>--<scenario-name>--<os-name>`

Can you use something like this please? It makes it easier to identify stuck machines on Hetzner later.